backend/capellacollab/sessions/routes.py

```diff
@@ -122,13 +122,16 @@ def request_session(
                 },
             )
         user = users.get_user(db, owner)
-        if user.role == users_models.Role.ADMIN:
-            repositories = [
-                repo.name for repo in repositories_crud.get_all_projects(db)
-            ]
-        else:
-            repositories = [repo.repository_name for repo in user.repositories]
-        session = OPERATOR.start_persistent_session(
+
+        # if user.role == users_models.Role.ADMIN:
+        #     repositories = [
+        #         repo.name for repo in repositories_crud.get_all_projects(db)
+        #     ]
+        # else:
+        #     repositories = [repo.repository_name for repo in user.repositories]
```

Comment:
```suggestion
        #FIXME: Get repositories from models with linked TeamForCapella repositories
```

I'd remove the commented lines as they can not really be reused. Instead, I'd add a FIXME comment so that fixes the behavior when it's possible to do so.